<launch>
  <node pkg="yac_ros" type="calib_vicon_validate_node" name="calib_vicon_validate_node" required="true" output="screen">
    <!-- <param name="rosbag_path" value="/data/vicon_capture.bag" /> -->
    <!-- <param name="calib_file" value="/tmp/vicon_calib.yaml" /> -->
    <param name="calib_file" value="/data/intel_d435i/calib_vicon_data/calib_results.yaml" />
    <param name="cam0_topic" value="/camera/color/image_raw" />
    <!-- <param name="body0_topic" value="/vicon/f450/pose" /> -->
    <param name="body0_topic" value="/vrpn_client_node/ucl_0/pose" />
  </node>
</launch>
